app/decorators/proposal_decorator.rb

```diff
@@ -55,4 +55,12 @@ def completed_status_message
   def progress_status_message
     "#{number_approved} of #{total_approvers} approved."
   end
+
+  def email_subject
+    "Request #{proposal.public_identifier}"
```

Comment:
Open to suggestions about this. The format suggested in [the story](https://www.pivotaltracker.com/story/show/93637304) is more likely to change during the lifetime of a request and thus more likely to break the threading...not sure what the right balance is. Some ideas, from most to least volatile/safe:

1. `Request #<ID>`
1. `Request #<ID> – <Proposal name>`
1. `Request #<ID>, <Org Code>, <Building code>, from <Requester>`